<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Aplikacja_geodezyjna</class>
 <widget class="QDialog" name="Aplikacja_geodezyjna">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Wprowadz_phi">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Podaj wartość phi w stopniach dziesiętnych</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Wprowadz_lambde">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Podaj wartość lambdy w stopniach dziesiętnych</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Wyswietlenie_wyniku_xy00">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Wartosc_phi">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Wartosc_lambdy">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_strefy">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="Strefa_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>5 --&gt; 15 stopni</string>
    </property>
   </widget>
   <widget class="QLabel" name="Strefa_u2000">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybierz strefę dla u2000</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="Strefa_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>6 --&gt; 18 stopni</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Strefa_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>7 --&gt; 21 stopni</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Strefa_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>8 --&gt; 24 stopnie</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_elipsoida">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="Elipsoida">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybierz elipsoidę odniesienia</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="GRS80">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>86</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>GRS80</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="WGS84">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>86</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>WGS84</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="xy2000">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Przelicz na xy2000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="xy_gk">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Przelicz na xy_gk00</string>
   </property>
  </widget>
  <widget class="QLabel" name="Wyswietlenie_wyniku_xy_gk">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>380</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Wybor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Wybierz, co chcesz zrobić</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Wyswietlenie_wyniku_xy92">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="xy92">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Przelicz na xy1992</string>
   </property>
  </widget>
  <widget class="QLabel" name="model1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="model2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="model3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
